client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -371,6 +372,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const buildYourOwnCta = isEnabled( 'signup/design-picker-build-your-own' ) && (
```

Comment:
I think we can create a new file for it and import it into the UDP directly rather than props. And if the selected theme is a blank canvas, we just go to the `buildYourOwnLayout` step. What do you think?

Reply:
@arthur791004 I tried to create new file for this but can't import the image in `client/assets/onboarding/blank-canvas.png` could you advice?

https://github.com/Automattic/wp-calypso/blob/61f1b0b695684945477e96d2e14ac21eb53f8f63/packages/design-picker/src/components/build-your-own-cta/index.tsx#L16

Reply:
Use `/calypso/images/onboarding/blank-canvas.png` for the image